ColorCoderTest.cpp

```diff
@@ -0,0 +1,34 @@
+#include "ColorCoderTest.h"
+
+    void testNumberToPair(int pairNumber,
+        TelCoColorCoder::MajorColor expectedMajor,
+        TelCoColorCoder::MinorColor expectedMinor)
+    {
+        TelCoColorCoder::ColorPair colorPair =
+            TelCoColorCoder::GetColorFromPairNumber(pairNumber);
+        std::cout << "Got pair " << colorPair.ToString() << std::endl;
+        assert(colorPair.getMajor() == expectedMajor);
+        assert(colorPair.getMinor() == expectedMinor);
+    }
+
+    void testPairToNumber(
+        TelCoColorCoder::MajorColor major,
+        TelCoColorCoder::MinorColor minor,
+        int expectedPairNumber)
+    {
+        int pairNumber = TelCoColorCoder::GetPairNumberFromColor(major, minor);
+        std::cout << "Got pair number " << pairNumber << std::endl;
+        assert(pairNumber == expectedPairNumber);
+    }
+
+    void DisplayColorPair()

```

Comment:
This is production code, but is in the test file? If someone wants the display code, they would be forced to take the test code as well...